CMakeLists.txt

```diff
@@ -164,7 +164,7 @@ set(${extension_name}_SOURCE_DIR "${CMAKE_BINARY_DIR}/${extension_name}")
 FetchContent_Populate(${extension_name}
  SOURCE_DIR     ${${extension_name}_SOURCE_DIR}
  GIT_REPOSITORY ${EP_GIT_PROTOCOL}://github.com/OpenwaterHealth/SlicerOpenLIFU.git
- GIT_TAG        9564cc8463734cd55683d3f90b61117758ff58c0
+ GIT_TAG        f672fbe59675da3bc441cf50a01486a982782541
```

Comment:
With this updated version of SlicerOpenLIFU I get a failed build because the `python-requirements.txt` location has moved since the previous version of SlicerOpenLIFU.

I think this path just needs to be updated: https://github.com/OpenwaterHealth/OpenLIFU-app/blob/f2949faa0452d8bc5a21ca3c9f50984fcc54835e/SuperBuild/External_python-sliceropenlifu-requirements.cmake#L16

Reply:
I wonder why it never gave me an error. I have updated the location now -  hopefully that fixes the issue!

Reply:
Yeah that's weird that it didn't give you an error! Actually can you check and verify that your custom app build has the python-requirements already installed upon build? Like the "install python reqs" button should say "_re_ install"

If not then that's a bug (unrelated to this PR) that the  python requirement install gets skipped on windows build

Reply:
It does say reinstall. I think it does install the requirements as expected. Maybe I didn't get the error cause I initially built the app with an older version of SlicerOpenLIFU . But I can do a fresh build just incase to double check.